packages/valory/skills/abstract_round_abci/tests/test_tools/test_rounds.py

```diff
@@ -61,8 +63,9 @@
 # this is how many times we need to iterate before reaching the last iteration for a base test.
 BASE_TEST_GEN_ITERATIONS = 4
 
-
-settings(deadline=5000)
+running_on_ci = os.getenv(CI)
+if running_on_ci:
+    settings.load_profile(CI)
```

Comment:
Using CI profile only on CI.

Reply:
aah here it is. So where is the env var set?

Reply:
See above.